test/e2e/server/server.test.ts

```diff
@@ -8,11 +8,10 @@ import { setupDatabase, setupServer } from '../utils/test-utils';
 describe('Server test', () => {
   let database: Sequelize;
   let server: FastifyInstance;
-  const channelId = 'a-new-channel';
 
   beforeAll(async () => {
     database = await setupDatabase(false);
-    server = setupServer(database);
+    server = setupServer();
```

Comment:
don't we need to shutdown server or something? I mean, it's saying that `server` is never used :thinking:

Reply:
The `server` warning is solved on [this pr ](https://github.com/input-output-hk/adrestia-multisig-coordination/pull/15)